packages/toolchain/elemental-cli/definition.yaml

```diff
@@ -1,6 +1,7 @@
 name: "elemental-cli"
-category: "utils"
-version: 0.0.1
+category: "toolchain"
+bin_name: "elemental"
```

Comment:
cant we just rename the binary in the elemental-cli side instead? So its simpler?

Reply:
we shall rename the repository, no ?

Reply:
nothing we should worry about for now, there is one elemental binary now.. a gui would probably go in another project, etc

Reply:
Opened https://github.com/rancher-sandbox/cOS-toolkit/issues/978 to track that separately